cantools/database/can/c_source.py

```diff
@@ -568,10 +568,7 @@
 '''
 
 INIT_SIGNAL_BODY_TEMPLATE_FMT = '''\
-    {type_name} tmp_{signal_name} = {signal_initial};
-    memcpy((void*)(&msg_p->{signal_name}),
-           (void*)&tmp_{signal_name},
-           sizeof({type_name}));
+    msg_p->{signal_name} = {signal_initial};
```

Comment:
if I did get that correctly, the values in these structures are stored using the [internal representation of the signal values](https://github.com/cantools/cantools/blob/master/tests/files/c_source/motohawk.h#L90) ...